string[] request
bool    was_the_last
---
bool    confirmation
